publish.gradle

```diff
@@ -14,6 +14,13 @@ static def getLocalPublicationTimestamp() {
     return date.format('yyyyMMddHHmmss')
 }
 
+static def getGitHash() {
+    String[] cmd = ['git', 'rev-parse', '--verify', 'HEAD'] // via https://stackoverflow.com/a/949391/2219998
+    def process = Runtime.getRuntime().exec(cmd)
+    process.waitFor()
+    return process.inputStream.getText()
+}
+
```

Comment:
We also have code like this in build.gradle of `support-base`, maybe both could call the same?
https://github.com/mozilla-mobile/android-components/blob/9d8bcc23c391e69876761262736a56c77e995a20/components/support/base/build.gradle#L11-L18

But not a blocker. :)

Reply:
I don't know enough about groovy to make this easy, especially because one uses the short arg and the other does not, and I'm already way outside the thing I'm supposed to be working on 😅 so I'll settle for comments that indicate that this is duplicated.

Reply:
I realized that trimming might be a good idea – I didn't notice the POM file looks like this currently:
```
    <tag>52ac53ba150cd7753b1dd0080689741eb82e4dd9
</tag>
```

So I'll add one more commit that does the trim.